src/references/supportedCurrencies.ts

```diff
@@ -10,7 +10,7 @@ export const supportedCurrencies = {
     label: i18n.t(i18n.l.settings.currency.ETH),
     mask: '[09999999999]{.}[999999999999999999]',
     placeholder: '0',
-    smallThreshold: 0.003,
+    smallThreshold: 0.000005,
```

Comment:
this `smallThreshold` value is in the `references` directory which is used across the app - currently, the `smallThreshold` is used in the coinlist so this will change the behavior of that (see: `buildCoinsList` line 63 in `helpers/assets`

Reply:
github didn't show me this before I hit squash... will revert and fix.

Reply:
we can create a new param for this other type of `smallThreshold`